ejb/src/main/java/org/wildfly/httpclient/ejb/HttpCancelHandler.java

```diff
@@ -75,6 +82,9 @@ protected void handleInternal(HttpServerExchange exchange) throws Exception {
         final String bean = parts[3];
         String invocationId = parts[4];
         boolean cancelIdRunning = Boolean.parseBoolean(parts[5]);
+
+        // process Cookies and Headers
+        // TODO: cancellation requires that a Cookie be present
```

Comment:
Is this incomplete?

Reply:
Yes. This is an edge case issue that I planned to return to later.